stable/rethinkdb/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+1. Get RethinkDB Driver Details
+
+NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+      Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "fullname" . }}-proxy'
+
+  echo Host: $(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }}-proxy -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
It would be good to have different instructions based on the `service.type` value, like we have in other charts